.github/workflows/check.yml

```diff
@@ -44,6 +44,21 @@ jobs:
           git fetch origin main --depth 1 && ./scripts/if-no-version-change.sh
           yarn version check
 
+  bump-version:
+    runs-on: ubuntu-latest
+
+    if: ${{ github.ref == 'refs/heads/main' }}
+
+    permissions:
+      contents: write
+
+    steps:
+      - name: Check for version bump
+        run: |
+          PACKAGE_VERSION=$(node -p -e "require('./package.json').version")
+          ./scripts/if-version-change.sh git tag v${PACKAGE_VERSION}
+          git push --tags
```

Comment:
We could make this push the one tag instead? `git push origin tag v${PACKAGE_VERSION}`? Not that this will have any other tags when running in CI, but eh, it jumped out at me.